### PR TITLE
Update 03-private-repo-package.md

### PR DESCRIPTION
**Pull request by graham73may**
## Summary

**[Add a Package from a Private Repository](https://pantheon.io/docs/guides/integrated-composer/private-repo-package)** - Adding GitLab instructions and syntax as it's slightly different from GitHub. 

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by whitneymeredith** [comment]
@dficker Can you please confirm this change is correct?

File: source/content/guides/integrated-composer/03-private-repo-package.md [hunk 1]
```diff
@@ -33,6 +37,16 @@ For this procedure, a GitHub token will be added to your code repository. It all
...
+    ```json:title=composer.json
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://oauth2:<token>@gitlab.com/mycompany/my-private-repo.git"
+        }
+    ],
+    ```
 
 1. Run the command below to require the package and specify the branch, prefixed with `dev-`
```
Comment (dficker) [inline source/content/guides/integrated-composer/03-private-repo-package.md:51]: "Run the command below" but there's no command here. There's sample lines of what should be modified in the "composer.json" file, but that's not a command.

It looks like the command should be this:

composer require mycompany/my-private-repo:dev-branch-name

**Review by dficker — COMMENTED** [review]
Hmm. Looks pretty good.

I guess I haven't used GitLab much, personally. I'd have to set up a GitLab account and a repository to verify this works, I guess. Or maybe I can find someone else to verify this works.

**Comment by whitneymeredith** [comment]
LGTM

**Comment by graham73may** [comment]
Also just noticed that these instructions are somewhat duplicated here: https://pantheon.io/docs/guides/integrated-composer

Is it with shortening the content in the link above to link to this page? Or I could submit a PR to include gitlab here as well?

**Pull request merged**